package/widevine/widevine.mk

```diff
@@ -13,6 +13,14 @@ WIDEVINE_DEPENDENCIES = protobuf wpeframework-clientlibraries host-python3
 
 WIDEVINE_INSTALL_STAGING = YES
 
+
+ifeq ($(BR2_PACKAGE_HAS_NEXUS),y)
```

Comment:
This issue is specific to - trying compile software widevine with kernel version < 4.16, not - trying to compile software widevine on nexus. If you're going to apply this match, I'd suggest using a condition like `if kernel version < 4.16`.